odkx-src/sync-endpoint-cloud-setup.rst

```diff
@@ -55,49 +54,47 @@ Setting up a DigitalOcean account
 Setting up a Droplet
 """""""""""""""""""""""""""""
 
-1. Use the `following link <https://www.python.org/downloads/>`_ in order to install the latest version of Python 3.0. Ensure that you are specifically installing an iteration of Python 3.0, as Python 2.0 will soon be deprecated. The installer should take about a minute to run.
+1. First, click on the :guilabel:`Create` dropdown button at the top right of the screen. Then, click on :guilabel:`Droplet` to create a droplet cloud server.
 
-  .. note::
-    If using Windows, make sure to download the Windows version of Python instead.
-
-2. Open a terminal or command line. Install module to manage DigitalOcean droplets, using command:
-
-  .. code-block:: console
-
-      $ pip3 install -U python-digitalocean
+  .. image:: /img/setup-digital-ocean/create-droplet.png
+   :width: 600
 
+2. In the Distributions tab, on the :guilabel:`Create Droplet` screen; select *18.04 (LTS) x64* under the Ubuntu dropdown. Next, choose a plan and data center region based on your needs.
+   
   .. note::
-    Windows users also have the option of either using PuTTY_, a free SSH client, in order to install the DigitalOcean module with pip. In the case that pip is not installed, Windows users can instead refer to the installation instructions from the following_ link and run the :file:`setup.py` file to install the module instead.
-
-  .. _PuTTY: https://www.chiark.greenend.org.uk/~sgtatham/putty/
-
-  .. _following: https://github.com/koalalorenzo/python-digitalocean#how-to-install
+    Sync Endpoint requires more than *2GB* of space to run, this means that plans below *4GB* will not work.
+  
+  .. image:: /img/setup-digital-ocean/do-distribution.png
+    :width: 600
 
+  .. image:: /img/setup-digital-ocean/do-plan.png
+    :width: 600
 
-3. Generate API token by logging into DigitalOcean console and clicking on :guilabel:`API` under the **MANAGE** section. Now, click on :guilabel:`Generate New Token` and enter a name.
+3. Scroll down to the :guilabel:`Select additional options`, click on the User data checkbox, copy and paste the contents of the :download:`cloud_init_DO.yml</files/cloud_init_DO.yml>` file in the text area provided.
```

Comment:
is this something you need to download? If so, suggest a "download, then copy and paste the contents."

Reply:
Once you click on the file, it opens in the browser. It doesn't need to be downloaded.